Sources/TuistSigning/SigningCipher.swift

```diff
@@ -0,0 +1,134 @@
+import Basic
+import CryptoSwift
+import Foundation
+import TuistCore
+import TuistSupport
+
+enum SigningCipherError: FatalError, Equatable {
+    case failedToEncrypt
+    case masterKeyNotFound(AbsolutePath)
+    case rootDirectoryNotFound(AbsolutePath)
+
+    var type: ErrorType { .abort }
+
+    var description: String {
+        switch self {
+        case .failedToEncrypt:
+            return "Encryption failed"
+        case let .masterKeyNotFound(masterKeyPath):
+            return "Could not find master.key at \(masterKeyPath.pathString)"
+        case let .rootDirectoryNotFound(fromPath):
+            return "Could not find root directory from \(fromPath.pathString)"
```

Comment:
What is `root` directory?

Reply:
Changed to `signingDirectory` as that makes more sense in this context